sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed an issue where batch subrequests would not authenticate properly if using `TokenCredential` authentication
+when the `BlobBatchClient` was created from `BlobContianerClient.GetBlobBatchClient`.
```

Comment:
did you accidentally add a new line after authentication?

Reply:
It was intentional as the line was getting too long for my tiny laptop screen and it doesn't actually put it on a new line in markdown. I can make it one line though.